packages/ui-components/src/components/Map/Table/Table.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import TableCell from '@material-ui/core/TableCell';
+import TableSortLabel from '@material-ui/core/TableSortLabel';
+import TableBody from '@material-ui/core/TableBody';
+import { StyledTable } from './StyledTable';
+import { FlexStart } from '../../Layout';
+import { useMapTable } from './useMapTable';
+
+export const Table = ({ serieses, measureData, className }) => {
+  const { getTableProps, getTableBodyProps, headerGroups, prepareRow, rows, columns } = useMapTable(
+    serieses,
+    measureData,
+  );
+
+  return (
+    <TableContainer className={className}>
+      <StyledTable {...getTableProps()} style={{ minWidth: columns.length * 140 + 250 }}>
+        <TableHead>
+          {headerGroups.map(({ getHeaderGroupProps, headers }) => (
+            <TableRow {...getHeaderGroupProps()}>
+              {headers.map(
+                ({ getHeaderProps, getSortByToggleProps, isSorted, isSortedDesc, render }) => (
+                  <TableCell {...getHeaderProps(getSortByToggleProps())}>
+                    <FlexStart>
+                      {render('Header')}
+                      <TableSortLabel active={isSorted} direction={isSortedDesc ? 'asc' : 'desc'} />
+                    </FlexStart>
+                  </TableCell>
+                ),
+              )}
+            </TableRow>
+          ))}
+        </TableHead>
+        <TableBody {...getTableBodyProps()}>
+          {rows.map(row => {
+            prepareRow(row);
+            return (
+              <TableRow {...row.getRowProps()}>
+                {row.cells.map(({ getCellProps, render }) => {
+                  return <TableCell {...getCellProps()}>{render('Cell')}</TableCell>;
+                })}
```

Comment:
```suggestion
                {row.cells.map(({ getCellProps, render }) => 
                  <TableCell {...getCellProps()}>{render('Cell')}</TableCell>;
                )}
```
Just to maintain consistency with the `Chart` version where possible